Week-1/Week.html

```diff
@@ -0,0 +1,142 @@
+            top: 0;
+            left: 0;
+            width: 50px;
+            height: 100vh;
+            background-color: rgb(190, 188, 188)
+        }
+        #image{
+            margin-left: 100px;
+            height: 200px;
+            width: 200px;
+            margin-top: 30px;
+        }
+        #title1{
+            font-family: 'Lato';
+            margin-top: 60px;
+            margin-left: 20px;
+        }
+        #info{
+            font-family: 'Lato';font-size: 12px;
+        }
+        table,th,tr{
+            font-family: 'Lato';font-size: 18px;
+        }
+        #dark_row{
+            background-color: rgb(168, 168, 168);
+            border-color: rgb(168, 168, 168);
+        }
+        #upper{
+            display: flex;
+            flex-direction: row;
+            background-color: rgb(57, 56, 56);
+            margin-top: 0%; 
+            width:100vw;
+            height: 35vh;
+        }
+        #bottom{
+            display: flex;
+            flex-direction: row;
+            width: 100vw;
+            height: 150px;
+            background-color:rgb(52, 101, 186);
+            position: fixed; bottom: 0; right: 0; left: 0;
+        }
+        #icon{
+            width:100px;
+            height:100px;
+            margin-top : 20px; 
+            margin-left: 20px;
+            
+        }
+    </style>
+</head>
+
+<body>
+    <div id="leftbar">
+            <i title="Menu" class="las la-bars" style="font-size:35px;margin-left: 5px; margin-top: 5px;margin-top:20px"></i> 
+            <i title="Home" class="las la-home" style="font-size:35px;margin-left: 5px; margin-top: 5px;margin-top:20px"></i> 
+            <i title="Search" class="las la-search" style="font-size:35px;margin-left: 5px; margin-top: 5px;margin-top:20px"></i> 
+            <i title="Volume-up" class="las la-volume-up" style="font-size:35px;margin-left: 5px; margin-top: 5px;margin-top:20px"></i> 
+            <i title="Profile" class="las la-user" style="font-size:35px;margin-left: 5px; margin-top: 5px;margin-top:20px"></i> 
+            <i title="Settings"class="las la-cog" style="font-size:35px;margin-left: 5px; margin-top: 5px;margin-top:20px"></i> 
+            <i title="Spotify"class="lab la-spotify" style="font-size:35px;margin-left: 5px; margin-top: 5px;margin-top:20px"></i> 
+            <i title="Sound-cloud"class="lab la-soundcloud" style="font-size:35px;margin-left: 5px; margin-top: 5px;margin-top:20px;"></i> 
+    </div>
+    <div id="upper">
+        <div>
+            <img id="image" src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg"; >
+        </div>    
+        <div id="title1"><h1 style="font-size: 22px;" >CattyBoard Top 100 singles chart (11.06.36)</h1>
+            <p id="info">Unknown Artist<br>
+                2016.Charts.100 Songs
+            </p>
+        </div>
+    </div>
+    <div>
+        <table style="width:90%;margin-left: 100px;">
```

Comment:
Its a good practice to not mix coding styles of adding CSS. Since the best approach is declaring classes, lets avoid using inline styles.